jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -143,25 +166,11 @@ pipeline {
                                 architecture: 'x64',
                                 distribution: 'rpm'
                             )
-                            String buildManifestUrl = buildManifestObj.getUrl(JOB_NAME, BUILD_NUMBER)
-                            String artifactUrl = buildManifestObj.getArtifactUrl(JOB_NAME, BUILD_NUMBER)
-                            env.ARTIFACT_URL_X64_RPM = artifactUrl
-
-                            echo "buildManifestUrl (x64, rpm): ${buildManifestUrl}"
-                            echo "artifactUrl (x64, rpm): ${artifactUrl}"
-
```

Comment:
no need to remove

Reply:
These are not used tho and we can add back later.

Reply:
Added now.